eth2/beacon/state_machines/forks/serenity/epoch_processing.py

```diff
@@ -296,14 +298,14 @@ def process_crosslinks(state: BeaconState, config: Eth2Config) -> BeaconState:
                 state, attesting_indices, crosslink_committee
             )
             if threshold_met:
-                new_current_crosslinks = update_tuple_item(
-                    new_current_crosslinks, shard, winning_crosslink
+                state = state.copy(
+                    current_crosslinks=update_tuple_item(
+                        new_current_crosslinks, shard, winning_crosslink
```

Comment:
Is it possible to use only `state.current_crosslinks`, and remove all other `new_current_crosslinks`?
```suggestion
                        state.current_crosslinks, shard, winning_crosslink
```